MapleServer2/Database/Classes/DatabaseShop.cs

```diff
@@ -10,6 +10,10 @@ public class DatabaseShop : DatabaseTable
         public Shop FindById(long id)
         {
             Shop shop = QueryFactory.Query(TableName).Where("Id", id).Get<Shop>().FirstOrDefault();
```

Comment:
you can use `.First()` to return a null if not found. i think you can.
```suggestion
            Shop shop = QueryFactory.Query(TableName).Where("Id", id).Get<Shop>().First();
```

Reply:
With first it will return ArgumentNullException or InvalidOperationException